samples/philosophers/sample.yaml

```diff
@@ -1,15 +1,26 @@
 sample:
   name: Dining Philosophers
```

Comment:
We could extend this more (maybe in another PR). Philosophers through ifdefs lets you use several different kernel object types for synchronization, i think we currently only test with mutexes IIRC.

Reply:
yes, good idea